google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -118,6 +120,11 @@ public void onOutputLine(String output) {
         deployLogListener,
         deployOutputListener,
         deployExitListener);
+
+    // show a warning notification if the cloud sdk version is not supported
+    new CloudSdkVersionCheck().performCloudSdkVersionCheck(
```

Comment:
I could just register this class as an applicationService as well - that would make it straightforward to use it in a more idiomatic way from here. What do you think about that?